pom.xml

```diff
@@ -2,9 +2,10 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>org.sonatype.oss</groupId>
-    <artifactId>oss-parent</artifactId>
-    <version>7</version>
+    <groupId>org.jenkins-ci</groupId>
+    <artifactId>jenkins</artifactId>
+    <version>1.62</version>
+    <relativePath />
```

Comment:
This will cease to publish to OSSRH and thus to Central. Probably OK for our purposes? I am not exactly sure what this plugin is for. If it is to appear in a plugin POM, then using our Artifactory only would be OK, since `pluginRepositories` should be defined.

Reply:
It's the thing that extracts licenses from dependencies to show on Jenkins about page: http://localhost:8080/jenkins/about/

it allows running groovy to filter licenses / choose one etc:

https://github.com/jenkinsci/jenkins/blob/e7ca5dbc4473206b7e90f8ea612493a04c57817c/war/pom.xml#L441
https://github.com/jenkinsci/jenkins/blob/master/licenseCompleter.groovy

Reply:
> This will cease to publish to OSSRH and thus to Central

yeah sounds fine